### DISCUSSION TITLE
Check for pre-release

### DISCUSSION THREAD

**Discussion by omidshojaee** [hiddify/hiddify-app#390] [Ideas]
Currently, "Check for update" checks for "Release" version.

It would be nice to have an option to check for "pre-release" versions too, and upgrade if needed.

**Comment by hossein935** [comment]
do you suggest me an application on IOS (iPhone) to work ?

**Comment by omidshojaee** [comment]
This suggestion is about the desktop client.

Allow checking for and downloading of the pre-release version of the client.
